### PR TITLE
feat: implement landing page

### PR DESCRIPTION
**Pull request by pavania1**
## Summary by CodeRabbit

- **New Features**
  - Introduced a new landing page with wallet connection functionality and custom particle animations.
  - Added a `WalletPopup` component for improved wallet interaction experience.

- **Enhancements**
  - Updated the governance page to streamline content display.
  - Enhanced user interface with new styles, including hover effects and visual feedback for wallet selection.

- **Refactor**
  - Simplified the import and usage of components across the platform for better maintainability.

- **Style**
  - Implemented a comprehensive update to global styles, affecting layout, color schemes, and animations for a more engaging user interface.

- **Documentation**
  - Updated public entity declarations to reflect the latest changes in components and functionality.

### CODE REVIEW

**Review by aleem1314 — CHANGES_REQUESTED** [review]
Device resolution: 1366 X 768
- Text overlaps image.
- Should be no vertical scroll on any screen size.
- Excess white space at the bottom of the footer.

landing
landing1

File: frontend/src/components/WalletPopup.tsx [hunk 1]
```diff
@@ -65,11 +65,11 @@
                   </div>
                 ))}
               </div>
-              <div>
+              {/* <div>
                 <button className="button">
                   <p className="popup-text">Connect</p>
                 </button>
-              </div>
+              </div> */}
```
[the remark this answers is not in the document]
Comment (pavania1) [reply frontend/src/components/WalletPopup.tsx:72]: maybe it is used for future implementation

File: frontend/src/components/LandingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        <div className="flex items-center gap-10 pl-20 py-0 ">
+          <div className="space-y-40">
+            <div className="flex flex-col space-y-6">
+              <div className="flex text-white text-center text-[100px] not-italic font-extrabold leading-[100px] tracking-[7px]">
+                Resolute
+              </div>
+              <div className="flex flex-col space-y-10">
+                <div className="flex flex-col space-y-2 w-[512px]">
+                  <div className="text-white text-[28px] font-light">
+                    Interface Interchain
```
Comment (aleem1314) [inline frontend/src/components/LandingPage.tsx:97]:
```suggestion
                    Interchain Interface
```

File: frontend/src/components/LandingPage.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+            />
+          </div>
+
+          <div className="absolute left-[521px] top-[38px]">
+            <Image
+              src="/background-circle.png"
+              width={402}
+              height={456}
+              alt="background-circle"
+            />
```
Comment (aleem1314) [inline frontend/src/components/LandingPage.tsx:126]: Write media queries for this image. Vertical scroll is appearing on smaller screens.

File: frontend/src/components/LandingPage.tsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+          </div>
+          <div className="flex justify-end w-full">
+            <div className="flex absolute  mt-20">
+              {/* controls loop muted autoPlay */}
+              <video width={802} height={10} controls>
+                <source src="/video.mp4" type="video/mp4" />
+              </video>
```
Comment (aleem1314) [inline frontend/src/components/LandingPage.tsx:133]: Why is the video still here? Didn't we decide to remove it?

File: frontend/src/components/LandingPage.tsx [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+            </div>
+            <Image
+              src="/landingpage.png"
+              width={967}
+              height={581}
```
Comment (aleem1314) [inline frontend/src/components/LandingPage.tsx:138]: Don't hard code image width and height. Write down media queries.

**Comment by aleem1314** [comment]
- Wallet buttons don't change when you hover.
- Hand cursor doesn't appear on hover.

File: frontend/src/components/LandingPage.tsx (hunk 4 above)
Comment (pavania1) [reply frontend/src/components/LandingPage.tsx:133]: i'm trying with ratio,  i think it would be work

File: frontend/public/insidelaptop.png
Comment (aleem1314) [inline frontend/public/insidelaptop.png:1]: Where are we using this image?

File: frontend/public/video.mp4
Comment (aleem1314) [inline frontend/public/video.mp4:1]: ditto here. Remove unnecessary files.

File: frontend/src/components/LandingPage.tsx [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+              isOpen={connectWalletDialogOpen}
+              onClose={handleClose}
+              selectWallet={selectWallet}
+            />
+          </div>
+
+          <div className="flex justify-end w-full z-10">
+            {/* <div className="flex absolute mt-10 ">
+              <Image src="/insidelaptop.png" height={550} width={820} alt="" />
+            </div> */}
```
Comment (aleem1314) [inline frontend/src/components/LandingPage.tsx:124]:
```suggestion
```

Remove commented code

File: frontend/src/components/WalletPopup.tsx [hunk 7]
```diff
@@ -65,11 +76,12 @@ const WalletPopup = ({
                   </div>
                 ))}
               </div>
+              {/* TODO: Re-enable this button in the future when the connect feature is implemented
```
Comment (aleem1314) [inline frontend/src/components/WalletPopup.tsx:79]: ditto here

File: frontend/src/components/LandingPage.tsx [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+                  <div className="text-white text-lg font-thin leading-normal">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
+                    do eiusmod tempor incididun.
+                  </div>
+                </div>
+                <div
+                  className="landingpage-button"
+                  onClick={() => setConnectWalletDialogOpen(true)}
+                >
+                  <p className="text-white text-lg font-bold z-10">
```
Comment (aleem1314) [inline frontend/src/components/LandingPage.tsx:107]: Add `z-index` to `connect wallet` button.

**Review by aleem1314 — APPROVED** [review]

**Pull request merged**